frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/operator.ts

```diff
@@ -0,0 +1,164 @@
+import { projectDropdown } from '@console/cypress-integration-tests/views/common';
+import { detailsPage } from '@console/cypress-integration-tests/views/details-page';
+import { listPage } from '@console/cypress-integration-tests/views/list-page';
+import { modal } from '@console/cypress-integration-tests/views/modal';
+import { nav } from '@console/cypress-integration-tests/views/nav';
+
+export const GlobalInstalledNamespace = 'openshift-operators';
+
+export const operator = {
+  install: (
+    operatorName: string,
+    operatorHubCardTestID: string,
+    installToNamespace: string = GlobalInstalledNamespace,
+  ) => {
+    cy.log(`install operator ${operatorName} in ${installToNamespace}`);
+    nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
+    cy.byTestID('search-operatorhub').type(operatorName);
+    cy.log('go to operator overview panel');
+    cy.byTestID(operatorHubCardTestID).click();
+    cy.log('go to the install form');
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+    if (installToNamespace !== GlobalInstalledNamespace) {
+      cy.log('configure Operator install form for single namespace');
+      cy.byTestID('A specific namespace on the cluster-radio-input').check();
+    } else {
+      cy.byTestID('All namespaces on the cluster-radio-input').should('be.checked');
+      // TODO some operators, such as 'Openshift Container Storage', do not support 'All namespaces' option and instead present a 'Operator recommended Namespace'
+      // cy.byTestID('Operator recommended Namespace:-radio-input').should('be.checked');
+    }
+    cy.log(`verify namespace dropdown shows the ${installToNamespace} namespace`);
+    cy.byTestID('dropdown-selectbox').should('contain', installToNamespace);
+
+    // Install
+    cy.byTestID('install-operator').click();
+    cy.log('verify Operator began installation');
+    cy.byTestID('view-installed-operators-btn').should(
+      'contain',
+      `View installed Operators in Namespace`,
+    );
+    cy.log(`navigate to OperatorHub in Namespace ${installToNamespace}`);
+    cy.byTestID('view-installed-operators-btn').click();
+    cy.byLegacyTestID('resource-title').should('contain', 'Installed Operators');
```

Comment:
This is still coupled with a successful install. The button you are looking for in these last steps will only appear after a successful installation. I would either move these into the test case or make a separate helper that verifies the successful in-progress steps of the installation.
```suggestion
  },
  installCompleted: () =>
    cy.log('verify Operator began installation');
    cy.byTestID('view-installed-operators-btn').should(
      'contain',
      `View installed Operators in Namespace`,
    );
    cy.log(`navigate to OperatorHub in Namespace ${installToNamespace}`);
    cy.byTestID('view-installed-operators-btn').click();
    cy.byLegacyTestID('resource-title').should('contain', 'Installed Operators');
```

Reply:
My mistake, that link shows immediately.